.github/workflows/ci2.yml

```diff
@@ -78,8 +78,8 @@ jobs:
         compiler: [clang, gcc]
         include:
           - distro: rocky
-            base: el.8
-            with: rockylinux/rockylinux:8
+            base: el.9
```

Comment:
Looks like your branch is based on an older version of feature/vos_on_blob but the branch has more recent commits to master.

You can resolve this and make this much easier to review if you merge latest feature/vos_on_blob changes with this branch.

Reply:
Yes, something went wrong with my rebase. I will rebase again

Reply:
BTW, the current feature/vos_on_blob doesn't build, multiple errors